lib/matplotlib/image.py

```diff
@@ -394,8 +394,8 @@ def _make_image(self, A, in_bbox, out_bbox, clip_bbox, magnification=1.0,
                 A_scaled = np.empty(A.shape, dtype=scaled_dtype)
                 A_scaled[:] = A
                 A_scaled -= a_min
-                a_min = a_min.astype(scaled_dtype)
-                a_max = a_max.astype(scaled_dtype)
```

Comment:
It took me a bit to understand why this was necessary (because ytarray inherits nparray).  Can you add a comment that makes it clear why this extra call is needed?  Otherwise looks fine to me!

Reply:
I've added the comment